modules/cas_cache/cas_cache.h

```diff
@@ -96,4 +97,63 @@ struct casdsk_functions_mapper {
 	struct gendisk *(*casdsk_exp_obj_get_gendisk)(struct casdsk_disk *dsk);
 };
 
+static inline int cache_name_from_id(char *name, uint16_t id)
+{
+	int result;
+
+	result = snprintf(name, OCF_CACHE_NAME_SIZE, "cache%d", id);
```

Comment:
snprintf does not return negative error code, insufficient buffer space is indicated by ret value >= len. Please change return value logic and add BUG_ON to assert enough buffer space (if overflow is not possible).

Reply:
I double checked snprinf() documentation and it says: "If an encoding error occurs, a negative number is returned.". I'll add `BUG_ON` for case when buffer length is not sufficient..

Reply:
Are you referring to the kernel variant? I see no information about negative error code:
https://elixir.bootlin.com/linux/latest/source/lib/vsprintf.c#L2616
https://www.kernel.org/doc/htmldocs/kernel-api/API-snprintf.html

Reply:
Done.